### PR TITLE
Ensure release sub-command can handle libraries:

### PR DESCRIPTION
**Pull request by robertvilla**
Added a list of digimap libraries which the get parameters function
can optionally search (i.e. if a release if being executed). In the
case of deploys the library list will not be checked.

Associated issue: https://github.com/edina/edina-hubot/issues/12

### CODE REVIEW

File: scripts/digimap-ci.coffee [hunk 1]
```diff
@@ -334,7 +350,7 @@ getValidUser = (res) ->
     res.reply "Error: User @#{user} is not permitted to release apps"
     return null
   else
-    return { user: user, token: token.token }
+  return { user: user, token: token.token }
```
Comment (agxs) [inline scripts/digimap-ci.coffee]: Is this whitespace correct? Coffeescript is whitespace sensitive like python is it not?

File: scripts/digimap-ci.coffee [hunk 2]
```diff
@@ -144,8 +149,19 @@ getParams = (args, res, callback) ->
         groupId = apps.services.groupId
         location = apps.services.location
 
+  # Check also libraries if alowed by the caller
+  if allowLibraries && app == null
+    for element in libraries.apps
```
Comment (agxs) [inline scripts/digimap-ci.coffee]: it's minor, but the `[...].filter` pattern is much nicer than the `for-if` pattern

File: scripts/digimap-ci.coffee [hunk 3]
```diff
@@ -30,7 +30,12 @@ apps =
   frontend:
     location: 'edina/digimap/'
     groupId: 'edina.digimap'
-    apps: ['cdptquery', 'codepoint', 'datadownload', 'digiadmin', 'gaz-plus', 'gaz-simple', 'interface', 'mapproxy', 'marinelexicon', 'roam', 'siterep', 'interface2']
+    apps: ['cdptquery', 'codepoint', 'datadownload', 'digiadmin', 'gaz-plus', 'gaz-simple', 'interface', 'mapproxy', 'marinelexicon', 'roam', 'siterep']
+
+libraries =
+  location: 'edina/digimap/'
+  groupId: 'edina.digimap'
+  apps: ['mapper-framework', 'user-persistence', 'authorisation', 'digimap-logging']
```
Comment (agxs) [inline scripts/digimap-ci.coffee]: `coordomatic` is used by some stuff.

File: scripts/digimap-ci.coffee (hunk 1 above)
Comment (robertvilla) [reply scripts/digimap-ci.coffee]: No, that's definitely not right. Will fix this

**Pull request merged**